data/19.0.0.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<release xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../schema.xsd" version="18.0.0">
+  <changelog href="https://nextcloud.com/changelog/#19-0-0"/>
+  <whatsNew lang="en">
+    <regular>
+      <item>Talk 9 integrates document viewing/editing and a grid view</item>
+      <item>Deck integration in Calendar - easily track your deadlines!</item>
```

Comment:
When not enabled the users can't do anything and don't even know what deck might be about?

Reply:
Same as with Talk - happy to advertise it, they can complain to the admin that they want it :rocket:

Reply:
That is actually not released in deck yet.

Reply:
But Talk is tightly bound to Nextcloud and installed on fresh setups. Not so Deck.

> That is actually not released in deck yet.

And this nails it :)

Reply:
Removed.